stories/rating-indicator/custom.scss

```diff
@@ -1,10 +0,0 @@
-.example-container {
```

Comment:
These should be reused in `custom.scss` inside `.storybook` folder instead of per component.